File: sample/Python/kabusapi_sendorder_cash_sell.py
import urllib.request
import json
import pprint

obj = { 'Password': '123456',
        'Symbol': '9433',
        'Exchange': 1,
        'SecurityType': 1,
        'Side': '1',
        'CashMargin': 1,
        'DelivType': 0,
        'FundType': '  ',
        'AccountType': 2,
        'Qty': 100,
        'FrontOrderType': 30,
        'Price': 2762.5,
        'ExpireDay': 0,
        'ReverseLimitOrder': {
                               'TriggerSec': 1, #1.発注銘柄 2.NK225指数 3.TOPIX指数
                               'TriggerPrice': 2600,
                               'UnderOver': 1, #1.以下 2.以上
                               'AfterHitOrderType': 1, #1.成行 2.指値 3. 不成
                               'AfterHitPrice': 0
                             }
        }
json_data = json.dumps(obj).encode('utf-8')

url = 'http://localhost:18080/kabusapi/sendorder'
req = urllib.request.Request(url, json_data, method='POST')
req.add_header('Content-Type', 'application/json')
req.add_header('X-API-KEY', 'ed94b0d34f9441c3931621e55230e402')

try:
    with urllib.request.urlopen(req) as res:
        print(res.status, res.reason)
        for header in res.getheaders():
            print(header)
        print()
        content = json.loads(res.read())
        pprint.pprint(content)
except urllib.error.HTTPError as e:
    print(e)
    content = json.loads(e.read())
    pprint.pprint(content)
except Exception as e:
    print(e)

File: sample/C#/CSharp/Kabusapi_Sendorder_Cash_sell.cs
using Newtonsoft.Json;
using System;
using System.Net.Http;
using System.Text;

namespace CSharp_sample
{
    class Kabusapi_Sendorder_Cash_sell
    {
        static void Main(string[] args)
        {
            var obj = new
            {
                Symbol = "9501",
                Exchange = 1,
                SecurityType = 1,
                Side = "1",
                CashMargin = 1,
                DelivType = 0,
                FundType = "  ",
                AccountType = 2,
                Qty = 100,
                FrontOrderType = 30,
                Price = 405,
                ExpireDay = 20210715,
                ReverseLimitOrder = new
                {
                    TriggerSec = 1,
                    TriggerPrice = 400,
                    UnderOver = 2,
                    AfterHitOrderType = 1,
                    AfterHitPrice = 0
                }
            };
            var url = "http://localhost:18080/kabusapi/sendorder";
            string token = GenerateToken.GetToken();

            try
            {
                var client = new HttpClient();
                var request = new HttpRequestMessage(HttpMethod.Post, url);
                request.Headers.Add("ContentType", "application/json");
                request.Headers.Add("X-API-KEY", token);
                request.Content = new StringContent(JsonConvert.SerializeObject(obj), Encoding.UTF8, "application/json");

                HttpResponseMessage response = client.SendAsync(request).Result;
                Console.WriteLine("{0} \n {1}", JsonConvert.DeserializeObject(response.Content.ReadAsStringAsync().Result), response.Headers);

            }
            catch (HttpRequestException e)
            {
                Console.WriteLine("{0} {1}", e, e.Message);
                Console.ReadKey();
            }
            catch (Exception ex)
            {
                Console.WriteLine("{0} {1}", ex, ex.Message);
                Console.ReadKey();
            }

            Console.ReadKey();
        }
    }
}
